docs/data-sources/elasticsearch_credential.md

```diff
@@ -0,0 +1,56 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "stackit_elasticsearch_credential Data Source - stackit"
+subcategory: ""
+description: |-
+  Manages Elasticsearch credentials
+---
+
+# stackit_elasticsearch_credential (Data Source)
+
+Manages Elasticsearch credentials
+
+## Example Usage
+
+```terraform
+resource "stackit_elasticsearch_instance" "example" {
+  name       = "example"
+  project_id = "example"
+  version    = "7"
+}
+
+resource "stackit_elasticsearch_credential" "example" {
+  project_id  = "example"
+  instance_id = stackit_elasticsearch_instance.example.id
+}
+
+data "stackit_elasticsearch_credential" "example" {
+  id          = stackit_elasticsearch_credential.example.id
+  project_id  = "example"
+  instance_id = stackit_elasticsearch_instance.example.id
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) Specifies the resource ID
```

Comment:
should data-source/es/credential.ID be read-only and not required like resource/es/credential.ID ?